git-edit-index

```diff
@@ -379,13 +382,17 @@ def parse_args(argv):
         action='version',
         version='%(prog)s {}'.format(__version__)
     )
-    parser.parse_args(argv)
+    parser.add_argument(
+        '--ignored',
```

Comment:
I suggest passing the following arguments into this call of `parser.add_argument()`:
* `help='...'` to describe what the option does (this will be printed when running the script with `--help`),
* `dest='ignored'` to make the name of the attribute more apparent (optional, just my suggestion).